pf9/cluster/cluster_attach.py

```diff
@@ -54,10 +54,11 @@ def cluster_exists(self, cluster_name):
     def wait_for_n_active_masters(self, cluster_name, master_node_num):
         TIMEOUT = 15
         POLL_INTERVAL = 30
-        timeout = int(time.time()) + (60 * TIMEOUT)
+        total_timeout_duration = 60 * TIMEOUT
```

Comment:
nit: can you add a comment on whether this interval is in minutes or seconds?

Reply:
Updated.